### ISSUE TITLE
Doesn't seem to be working

### ISSUE THREAD

**Issue by ovladuk** [brian-courts/clipper2#2]
when i click save it doesn't do anything. has this stopped working or is it just my end? also does it work on network streams?

**Comment by brian-courts** [comment]
I'm sorry it's not working for you. 
I don't have access to my computer right now. I'll get back to you when I have a chance. 
 
  On Thu, Jun 9, 2022 at 7:00 PM, ***@***.***> wrote:   


when i click save it doesn't do anything. has this stopped working or is it just my end? also does it work on network streams?

**Comment by ovladuk** [comment]
i figured it out. i didnt have ffmpeg in the VLC root folder.

**Comment by ovladuk** [comment]
is it possible to clip network videos? cause you can stream youtube videos straight to vlc would be good to be able to clip them.

**Comment by brian-courts** [comment]
I'm glad you figured it out. I hope Clipper2 serves you well.
 
 
  On Fri, Jun 10, 2022 at 9:16 AM, ***@***.***> wrote:   


i figured it out. i didnt have ffmpeg in the VLC root folder.

**Comment by brian-courts** [comment]
Clipper2 just sends a command to ffmpeg based on the parameters you set.
I see in the ffmpeg documentation that it can deal with stream input.  I don't know how vlc would determine the url of the stream or how a user would set the start and stop times of the clip.  
 
  On Fri, Jun 10, 2022 at 9:32 AM, ***@***.***> wrote:   


is it possible to clip network videos? cause you can stream youtube videos straight to vlc would be good to be able to clip them.

**Comment by silentpatrik** [comment]
> Clipper2 just sends a command to ffmpeg based on the parameters you set. I see in the ffmpeg documentation that it can deal with stream input.  I don't know how vlc would determine the url of the stream or how a user would set the start and stop times of the clip.  On Fri, Jun 10, 2022 at 9:32 AM, ***@***.***> wrote: is it possible to clip network videos? cause you can stream youtube videos straight to vlc would be good to be able to clip them. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>

No, its not so simple, unfortanly. Example, running ubuntu and having a mounted remote sftp location is like, `/run/user/1000/gvfs/sftp:host=192.168.1.109,user=networkuser/media` 

When the ffmpeg command is runned, here is how the path looks like:
`run/user/1000/gvfs/sftp%3Ahost%3D192.168.1.109%2Cuser%3Dnetworkuser/media`

If i manually copy the line and  replace %3a with ":" and %3d with =, "%2c"," and paste in terminal, prefixed by a slash (that is removed somewhere in plugin) all works perfect. 
But as of now, the above gives an error directory doesnt exists, and, of course it also should since it does not exist.

It was 10 years since i wrote any lua but i managed to fix the initial first slash, by adding this to ~line 605.
```
isnetwork = string.find(source_path, 'sftp:')
	if (found > 0 and isnetwork==nil ) then 
```

But i would need some help to find where the urlencoding is done. Help me and i will fix and test and make a pr asap.



BR
